trellis-http/src/test/java/org/trellisldp/http/AbstractLdpResourceTest.java

```diff
@@ -200,6 +201,14 @@
         allModes.add(ACL.Control);
         allModes.add(ACL.Read);
         allModes.add(ACL.Write);
+
+        allInteractionModels.add(LDP.Resource);
```

Comment:
If we have Guava on line (I think we do) maybe use something from `Sets` here to construct and fill the set?

Reply:
Guava isn't a `compile` dependency, but I have no problem with adding it as a test dependency (if it's not already there).